src/test/java/bio/terra/common/fixtures/ConnectedOperations.java

```diff
@@ -843,14 +887,67 @@ public MockHttpServletResponse retrieveSnapshotPreviewByIdRaw(
     if (filter != null) {
       request.param("filter", filter);
     }
+    if (sort != null) {
+      request.param("sort", sort);
+    }
     MvcResult result = mvc.perform(request).andReturn();
     return result.getResponse();
   }
 
+  public enum TDRResourceType {
```

Comment:
Nit: I think we like to follow Google's style guide, which uses camel case in names including for acronyms.  Under that scheme, this would be `TdrResourceType`. https://google.github.io/styleguide/javaguide.html#s5.3-camel-case